packages/backend/src/build-disk-image.spec.ts

```diff
@@ -54,6 +57,27 @@ vi.mock('@podman-desktop/api', async () => {
     configuration: {
       getConfiguration: (): Configuration => config,
     },
+    window: {
+      withProgress: vi.fn(),
+      showErrorMessage: vi.fn(),
+    },
+    ProgressLocation: {
+      TASK_WIDGET: 'TASK_WIDGET',
+    },
+  };
+});
+
+vi.mock('./container-utils', async () => {
+  return {
+    getContainerEngine: vi.fn().mockReturnValue({
+      connection: { type: 'podman', status: vi.fn().mockReturnValue('started') },
+    } as unknown as ProviderContainerConnection),
```

Comment:
The mockReturnValue should be done in the beforeEach, running `vi.resetAllMock()` remove all the mockReturnValue.

Reply:
if the beforeEach(() => { vi.resetAllMock() }) is missing it should be added

Reply:
Switched to using resetAllMock(), but since only the one new test needs getContainerEngine() to be mocked I moved the mocking into the test. (and found I don't actually need to mock machine-utils return value, just need basic mock()).